patches/nginx-1.11.2-privilege_process.patch

```diff
@@ -0,0 +1,232 @@
+diff --git a/src/core/nginx.c b/src/core/nginx.c
+index 60f8fe7..b15f84c 100644
+--- a/src/core/nginx.c
++++ b/src/core/nginx.c
+@@ -53,6 +53,13 @@ static ngx_command_t  ngx_core_commands[] = {
+       offsetof(ngx_core_conf_t, master),
+       NULL },
+ 
++    { ngx_string("privilege_process"),
```

Comment:
good idea, its more flexible and we can easily control it at Lua land. How about `ngx.worker.privilege({enable=true})` ? and It only can be runned at `init_by_lua` phase.